### DISCUSSION TITLE
Image | Orange Pi Zero

### DISCUSSION THREAD

**Discussion by thomas-pegot** [MichaIng/DietPi#6000] [Image Request]
Hello,

Is it possible to have the OPI Zero H2+ (ArmV7)  image ?

Thanks

**Comment by Joulinar** [comment]
let me link an older request for Orange Pi Zero2 https://github.com/MichaIng/DietPi/issues/4309#issuecomment-834238817
At the moment we don't have the capacity to create images for Orange Pi.

**Reply by thomas-pegot** [reply]
So the solution is to launch from debian : https://github.com/MichaIng/DietPi/blob/master/.build/images/dietpi-installer

**Reply by Joulinar** [reply]
yes. that's the starting point.

**Comment by lordslair** [comment]
Sorry to bump such an old discussion, but I've come across a few of these cards, and I'd love to try to have Dietpi on them

I'm going to look into the install script
If at some point I got a successful something, is there a way to PR/help to build related images for this hardware ?

**Comment by MichaIng** [comment]
Let me get some clarification: This is about the "Orange Pi Zero", which can have a H2 and a H3 SoC, based on revision, it seems, right?

http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-Zero.html

Because I cannot find the existence of an "Orange Pi Zero H2+".

EDIT: Yeah, just found the Armbian config for it, which mentions "H2+" as SoC: https://github.com/armbian/build/blob/main/config/boards/orangepizero.csc

So that should be pretty easy to implement, have a look at our recent SBC additions: https://github.com/MichaIng/DietPi/pull/7777/changes

**Reply by lordslair** [reply]
To give more inputs about the SoC
Physically on the hardware, the chip is labeled as H2+ (at least on the boards i have)

And you're right Orange Pi lists only for Zero:
• H2 Quad-core Cortex-A7 H.265/HEVC 1080P.
• H3 Quad-core Cortex-A7 H.265/HEVC 1080P

So just to give some details from what I've read
```
H2+ is a variant of H3, targeted at low-end OTT boxes, which lacks Gigabit MAC and 4K HDMI output support. 
So, H2+ is cost-down version of already cheap H3
```

I'm waiting for some cables to connect to the serial and the video output.
I'll try installing a basic debian `armhf-netinst.iso` to start with
